UndertaleModTool/Windows/FindReferencesTypesDialog/UndertaleResourceReferenceMethodsMap.cs

```diff
@@ -624,7 +626,7 @@ IEnumerable<object[]> GetExtnFunctions()
                             if (objSrc is not UndertaleString obj)
                                 return null;
 
-                            var codeLocals = data.CodeLocals.Where(x => x.Name == obj || x.Locals.Any(l => l.Name == obj));
+                            var codeLocals = data.CodeLocals?.Where(x => x.Name == obj || x.Locals.Any(l => l.Name == obj));
```

Comment:
```suggestion
                            var codeLocals = data.CodeLocals.Where(x => x.Name == obj || x.Locals.Any(l => l.Name == obj));
```
Please revert this change. If something goes wrong the error shouldn't  just be swallowed here, especially after #1947 fixes the related error handling.